### ISSUE TITLE
Question about symmetries

### ISSUE THREAD

**Issue by smart-fr** [jonathan-laurent/AlphaZero.jl#32]
Hello!
Not an issue but a question regarding GameInterface.symmetries (cf [the documentation](https://jonathan-laurent.github.io/AlphaZero.jl/dev/reference/game_interface/#AlphaZero.GameInterface.symmetries)):
> `symmetries(::Type{G}, state) where {G <: AbstractGame}
> `
> Return the vector of all pairs (s, σ) where:
> - s is the image of state by a nonidentical symmetry
> - σ is the associated actions permutation, as an integer vector of size num_actions(Game).

When applying to the game `g` with state `state1` and actions mask `actions_mask1` the symmetry corresponding to a given `(state2, σ)` pair, I suppose AlphaZero.jl affects `state2` as the new state of `g` and determines the updated actions mask `actions_mask2` according to one of the following propositions, but which one?
1. `actions_mask2[action_index] == actions_mask1[σ(action_index)]`
2. `actions_mask2[σ(action_index)] == actions_mask1[action_index]`

In other words, how is the permutation supposed to be used?
(Does it really matters in the end? ;-)

**Comment by jonathan-laurent** [comment]
Very good question! I should disambiguate in the documentation indeed.
The answer is 1.

**Comment by smart-fr** [comment]
Thank you!
